src/lib/Probabilities.ts

```diff
@@ -0,0 +1,17 @@
+import jStat from 'jstat'
+
+// Probability Distributions
+export class FakeNegativeBinomial {
```

Comment:
Extracted the fake negative binomial from `simulation.js` to its own file so I can import it. No real change here.